### PR TITLE
Feature/entity fix

### PR DESCRIPTION
**Pull request by borkarsaish65**

### CODE REVIEW

File: module/observations/helper.js [hunk 1]
```diff
@@ -1876,13 +1879,22 @@ module.exports = class ObservationsHelper {
...
+                  entityType: solutionData.data.entityType,
+                };
+          
+                let entitiesDocument = await entityManagementService.entityDocuments(
+                  filterData
+                );
+
+                if(!entitiesDocument.success){
+                  throw new Error({
+                    message:'Entity Not found.'
```
[the remark this answers is not in the document]
Comment (borkarsaish65) [reply module/observations/helper.js:1893]: Added

File: module/observations/helper.js [hunk 2]
```diff
@@ -1963,12 +1971,23 @@ module.exports = class ObservationsHelper {
         let entities = [];
 
         if (observationDocument[0].entities && observationDocument[0].entities.length > 0) {
-          let entitiesData = await entitiesHelper.entityDocuments(
-            {
-              _id: { $in: observationDocument[0].entities },
-            },
-            ['metaInformation.externalId', 'metaInformation.name'],
-          );
+          let entitiesData;
```
[the remark this answers is not in the document]
Comment (borkarsaish65) [reply module/observations/helper.js:1974]: added

File: module/observations/helper.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (VISHNUDAS-tunerlabs) [reply module/observations/helper.js:1893]: @borkarsaish65 Entity Not found.' also should come from a api response variable

[the remark this answers is not in the document]
Comment (borkarsaish65) [reply module/observations/helper.js]: Done

**Pull request merged**